wfe/web-front-end_test.go

```diff
@@ -436,7 +437,12 @@ func TestIssueCertificate(t *testing.T) {
 	// TODO: Use a mock RA so we can test various conditions of authorized, not
 	// authorized, etc.
 	stats, _ := statsd.NewNoopClient(nil)
-	ra := ra.NewRegistrationAuthorityImpl(fakeClock, wfe.log, stats)
+	ra := ra.NewRegistrationAuthorityImpl(fakeClock, wfe.log, stats, cmd.RateLimitConfig{
+		TotalCertificates: cmd.RateLimitPolicy{
```

Comment:
Why does the RateLimitConfig need to be defined here? The zero value of a RateLimitConfig should be a useful object, i.e. one with no rate limits enabled?